runtime/src/constants.rs

```diff
@@ -2,9 +2,11 @@ pub mod currency {
 	pub use primitives::Balance;
 
 	pub const HDX: Balance = 1_000_000_000_000;
-	pub const DOLLARS: Balance = HDX / 10;	// 10 HDX ~= 1 $
+	pub const DOLLARS: Balance = HDX / 10; // 10 HDX ~= 1 $
 	pub const CENTS: Balance = DOLLARS / 100;
 	pub const MILLICENTS: Balance = CENTS / 1_000;
+
+	pub const FORTUNE: Balance = u128::MAX;
```

Comment:
Why do we have this here? (Didn't find it used anywhere)

Reply:
Meant to be used for disabling certain features (e.g. set for minimum candidacy bond to council)

Reply:
What I meant as an example, is a prevention for other accounts to submit their candidacy to council